src/components/data-management/ProjectsListContainer.jsx

```diff
@@ -1,153 +1,47 @@
-import React, { useState, useEffect } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import PropTypes from 'prop-types';
+import React, { useState, useCallback } from 'react';
 import {
-  Card, Space, Descriptions, Skeleton,
+  Space, Tooltip, Input, Skeleton,
 } from 'antd';
-import { blue } from '@ant-design/colors';
-import EditableField from '../EditableField';
-
-// eslint-disable-next-line import/no-extraneous-dependencies
+import { useSelector } from 'react-redux';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
 
-import ProjectDeleteModal from './ProjectDeleteModal';
-import { setActiveProject, updateProject, deleteProject as deleteProjectAction } from '../../redux/actions/projects';
-import PrettyTime from '../PrettyTime';
-import validateInputs, { rules } from '../../utils/validateInputs';
-import integrationTestConstants from '../../utils/integrationTestConstants';
+import ProjectsList from './ProjectsList';
 
 const ProjectsListContainer = (props) => {
   const { height } = props;
-  const dispatch = useDispatch();
 
   const loading = useSelector((state) => state.projects.meta.loading);
-  const projects = useSelector((state) => state.projects);
-
-  const { activeProjectUuid } = projects.meta;
-  const [deleteModalVisible, setDeleteModalVisible] = useState(false);
-  const [deleteProjectUuid, setDeleteProjectUuid] = useState(false);
-  const [projectNames, setProjectNames] = useState(new Set());
-
-  useEffect(() => {
-    setProjectNames(new Set(projects.ids.map((id) => projects[id].name.trim())));
-  }, [projects.ids]);
-
-  const activeProjectStyle = {
-    backgroundColor: blue[0],
-    cursor: 'pointer',
-    border: `2px solid ${blue.primary}`,
-  };
-
-  const deleteProject = () => {
-    dispatch(deleteProjectAction(deleteProjectUuid));
-    setDeleteModalVisible(false);
-  };
-
-  const validationChecks = [
-    rules.MIN_8_CHARS,
-    rules.MIN_2_SEQUENTIAL_CHARS,
-    rules.ALPHANUM_DASH_SPACE,
-    rules.UNIQUE_NAME_CASE_INSENSITIVE,
-  ];
+  const [filterParam, setFilterParam] = useState('');
 
-  const validationParams = {
-    existingNames: projectNames,
-  };
+  const debouncedSetFilterParam = useCallback(
+    _.debounce((value) => {
+      setFilterParam(new RegExp(value, 'ig'));
```

Comment:
what's 'ig'?

Reply:
It's the flags for Regex : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/RegExp

i : case insensitive search
g : global search - find all matches instead of stopping at the first one

Reply:
I think the `g` flag can be removed as we only need the first hit, not every occurence of the string. It'll help speed up the searching. I'll change this.